Banjo Adjustments

CSharpM7 opened the pull request:
- WIP Some small tweaks to his overall strengths, nothing massive but still noteworthy


## Dash Attack:
Decreased start-up to be more similar to DK's and more inline with the animation
- [+] Active Frames: 10-14,14-22 -> 8-12,12-20
- [/] Automatically transitions to fall state after frame 41 if airborne

## UpTiit:
Decreased KB because the move was too fast and too strong
Banjo Hitbox:
- [-] KBG: 88/84->81/74
- [-] BKB: 81->74

## DTilt:
Slightly pulled-in hitboxes on later frames to better match animation. A minor nerf to edgeguarding
Old DTilt hitboxes:
https://imgur.com/a/4CsrUsY
New Dtilt hitboxes:
https://imgur.com/a/Wlm3nTs
- [-] Hitbox pulled inward by 1.5 units from frames 11-17

## Up Smash:
- [+} KBG: 110->122

## Nair:
Nerfs attempt to fix issues related to reliably hitting shield thrice, though a new animation might have to be made to solve that issue, as well as its incredibly strong knockback. 
Old Nair Hitboxes:
https://imgur.com/a/7cvfUqK
New Nair Hitboxes:
https://imgur.com/a/XIWIIIm

- [/] Sourspot hitbox is now attached to Banjo's right hand
- [-] Similar to other tippers, the backpack now takes less priority over the sourspot
- [-] Damage: 13/11,10/9,7->12/10,9/7,7
- [-] Sweetspot Active Frames: 8-13,15-20,25->28 -> 8-12,15->20,25->28
- [-] Sourspot Active Frames: 8-13,15-20 -> 8-12,12->20
- [-] BKB: (88/92, 86/90, 78/93) -> 48,48,78 
- [-] KBG: (48/46, 55/60, 26/43) -> 73,76,26

## Dair:
New Hitbox/HurtBox:
![2023090623405008](https://github.com/HDR-Development/HewDraw-Remix/assets/13909643/916f1690-1bf4-4b0a-bc50-3fb1349592d0)

Decreases advantage upon bouncing on shield, and overall disjoint of move
- [+] Starts 1 frame earlier on frame 14
- [-] Landing hitbox removed
- [-] Dair Bounce FAF increased by 6 frames
- [-] From frames 6-53, Kazooie's hurtbox become active (greatly lowering the overall hurtbox on Banjo)
- [-] Hitbox lower point shifted up by 2 units, now uses interpolation
- [/] Speedlines removed after bouncing, added feather scatter effect on bounce

## Wonderwing:
Old Hitbox:
![OldHitbox](https://github.com/HDR-Development/HewDraw-Remix/assets/13909643/cc4bda3f-4539-4be8-88cd-0fa7cd2d33b8)
New Hitbox:
![2023090622461254](https://github.com/HDR-Development/HewDraw-Remix/assets/13909643/70f14b5f-1967-440c-836c-43c7e984da67)
Decreases edgeguarding power with this move slightly, just start using dtilt and downsmash instead
- [-] Hitbox moved up by 4.0 units, converted to interpolated sphere

## Beakbomb:
A strong hit to this incredible recovery move, this move becomes slightly more risky to use as a recovery tool as once you get hit, you won't be getting it back until you land!
- [-] Removed startup armor
- [-] Strong hitbox active frames: 18->13 (weaker hitbox begins after this frame)
- [-] Similar to other OncePerAirtime Specials, cooldown is set upon entering side special (as opposed to activating during the dash)
- [/] Fixed an issue where Banjo could transition from Beakbomb to Wonderwing on certain frames
- [-] Cooldown will start...cooling down...when Banjo first touches the ground (rather than upon exiting Side Special)

CSharpM7 committed:
Special S Cleanup

-No more armor
-Should no longer transition from air-to-ground or vice versa, keeping Wonderwing and Beakbomb seperate

CSharpM7 committed:
Banjo uptilt nerf, upsmash buff

Utilt: Banjo BKB 81->74, KBG 88/84->81/74

CSharpM7 committed:
temporarily fix nair triple hit, also bkb nerf

BKB: (88/92/86/90.78/93) -> 48/78 (popup hitbox))
KBG: (48/46/55/60/26/43) -> 73/76/26(popup hitbox)

CSharpM7 committed:
Removed Dair Landing Hitbox

CSharpM7 committed:
more nair changes

Removed 3rd body hitbox
Damage: 13/11/10/9/7->12/10/9/7/7
Backpack sweetspot now has less priority than body hitbox
Body sourspot now tied to leg. Active Frames: 8-13,15-20 -> 8-13,13-18

CSharpM7 committed:
begin cooldown instantly

CSharpM7 committed:
Merge branch 'dev' into banjo-balance

CSharpM7 committed:
we do some compiler error fixing

CSharpM7 committed:
Update status.rs

SkewedAskew reviewed (dismissed):
- Dair somehow has no acmd running, it doesn't stall and doesn't have a hitbox...?
- Performing Beak Bomb right near a ledge and then landing immediately makes Banjo T-pose
- Landing during Beak Bomb gives Banjo a very big slide

CSharpM7 committed:
Merge branch 'dev' into banjo-balance

CSharpM7 committed:
Fixed bugs, slight balance changed

Dair: Fixed acmd. Dair bounce increased FAF by 6 frames
Side Special: Fixed tposing issue, fixed extreme sliding from crashing on the ground

CSharpM7 committed:
DTilt, Nair, WW and BB

Dtilt/Nair: Hitbox adjustments

WW:
- [-] Lower hitbox moved up by .75 units

BB:
- [-] Strong hitbox active frames: 18->13 (weaker hitbox begins after this frame)

CSharpM7 committed:
more adjustments

CSharpM7 committed:
dair nerfs

CSharpM7 committed:
effect change

CSharpM7 committed:
Update status.rs

CSharpM7 committed:
Merge branch 'dev' into banjo-balance

CSharpM7 committed:
Update status.rs
